src/iframe.html

```diff
@@ -1,16 +1,29 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <link href='http://fonts.googleapis.com/css?family=Ubuntu:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="styles/main.min.css">
-    <link rel="stylesheet" href="https://cartodb-libs.global.ssl.fastly.net/cartodb.js/v3/3.15/themes/css/cartodb.css" />
+    <link
+      href="http://fonts.googleapis.com/css?family=Ubuntu:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400..700;1,400..700&family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Public+Sans:ital,wght@0,900;1,900&display=swap"
```

Comment:
Same here as the [other comment](https://github.com/GreenInfo-Network/seattle-building-dashboard/pull/130/files#r1868523858). Formatting changes make this look like a lot happened, but just switching to Open Sans.